neo-gui/UI/NetFeeDialog.cs

```diff
@@ -17,18 +17,15 @@ public partial class NetFeeDialog : Form
         Fixed8 NetFee;
         Fixed8 PriorityFee;
```

Comment:
Is this var used now?

Reply:
@shargon 
`Fixed8 PriorityFee;` Not used now. I will remove it.

Thanks.